JsonLedgerApiSample/src/main/java/com/example/signing/TopologySignatureProvider.java

```diff
@@ -0,0 +1,14 @@
+package com.example.signing;
+
+import com.example.client.ledger.model.Signature;
+import com.google.protobuf.InvalidProtocolBufferException;
+
+import java.security.InvalidKeyException;
+import java.security.KeyPair;
+import java.security.NoSuchAlgorithmException;
+import java.security.SignatureException;
+import java.util.List;
+
+public interface TopologySignatureProvider {
+    Signature sign(KeyPair keyPair, List<String> transactions, String transactionMultiHash) throws NoSuchAlgorithmException, SignatureException, InvalidKeyException, InvalidProtocolBufferException;
```

Comment:
A comment or name that clarifies that transactions are base64 encoded would be helpful.